middleware/csrf/config.go

```diff
@@ -89,6 +94,12 @@ var ConfigDefault = Config{
 	CookieSameSite: "Strict",
 	Expiration:     1 * time.Hour,
 	KeyGenerator:   utils.UUID,
+	ErrorHandler:   DefaultErrorHandler,
+}
+
+// Default ErrorHandler that process return error from fiber.Handler
+var DefaultErrorHandler = func(c *fiber.Ctx, err error) error {
```

Comment:
It's better to make `DefaultErrorHandler` private.

Reply:
@kiyonlin 
Thank you for review.
I fixed. 1ad5a618cb0da239c41a218d3b06029556e7e571